src/wallet.ts

```diff
@@ -335,7 +335,7 @@ export class Wallet {
         account_id: MARKET_ACCOUNT,
         msg: JSON.stringify({
           price: price,
-          autotransfer: autotransfer || true,
```

Comment:
if sent in false, it would always be true, (no false = true)

Reply:
👍